File: POOUber/python/UberVan.py
from car import Car

class UberVan(Car):
    typeCarAccepted = []
    seatsMeterial = []

    def __init__(self, license, driver, typeCarAccepted, seatsMaterial):
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMeterial = seatsMaterial

File: POOUber/js/uberVan.js
class UberVan extends Car{
    constructor(license, driver, typeCarAcceppted, seatsMaterial) {
        super(license, driver);
        this.typeCarAcceppted = typeCarAcceppted;
        this.seatsMaterial = seatsMaterial;
    }

    get passenger(){
        return this._passenger;
    }

    set passenger(passengers){
        if(passengers == 6){
            this._passenger = passengers;
        } else {
            console.log("Debes tener 6 pasajeros");
        }
    }

}